velero-plugins/route/restore_test.go

```diff
@@ -15,3 +18,57 @@ func TestRestorePluginAppliesTo(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, velero.ResourceSelector{IncludedResources: []string{"routes"}}, actual)
 }
+
+func TestRestorePlugin_Execute(t *testing.T) {
+	restorePlugin := &RestorePlugin{Log: test.NewLogger()}
+
+	testcase := map[string]struct {
+		route routev1API.Route
+		want  string
+	}{
+		"empty": {route: routev1API.Route{Spec:
+		routev1API.RouteSpec{Host: ""},
+		}, want: ""},
+		"true": {route: routev1API.Route{ObjectMeta: metav1.ObjectMeta{
+			Annotations: map[string]string{
+				"openshift.io/host.generated": "true",
+			},
+		},
+			Spec:
+			routev1API.RouteSpec{Host: "test"},
+		}, want: ""},
+		"static": {route: routev1API.Route{ObjectMeta: metav1.ObjectMeta{
+			Annotations: map[string]string{
+				"openshift.io/host.generated": "static",
```

Comment:
For this test just don't have the annotation. Don't think `static` is a valid value here

Reply:
changed by removing static in the annotation and testing host.spec